### PR TITLE
correctly source lootbox volume

### PR DESCRIPTION
**Pull request by eli-d**
- pass `LogIndex` through winners stack
- add `LogIndex` to user actions
- use log index and transaction hash to look up a win's corresponding send transaction in `microservice-ethereum-create-transaction-lootboxes`

### CODE REVIEW

File: database/102-up-timescale/20230502151630-pending_reward_type_add_log_index.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+-- migrate:up
+
+ALTER TABLE ethereum_pending_reward_type
+	ADD COLUMN log_index BIGINT;
```
Comment (af-afk) [inline database/102-up-timescale/20230502151630-pending_reward_type_add_log_index.sql]: Should be DEFAULT and to be not null

File: database/102-up-timescale/20230502154725-user_actions_add_log_index.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+-- migrate:up
+
+ALTER TABLE user_actions
```
Comment (af-afk) [inline database/102-up-timescale/20230502154725-user_actions_add_log_index.sql:3]: NOT NULL and DEFAULT

File: database/102-up-timescale/20230502151630-pending_reward_type_add_log_index.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+-- migrate:down
+
+ALTER TABLE ethereum_pending_reward_type
+	DROP COLUMN log_index BIGINT;
```
Comment (eli-d) [inline database/102-up-timescale/20230502151630-pending_reward_type_add_log_index.sql]: remove BIGINT

File: database/102-up-timescale/20230502151630-pending_reward_type_add_log_index.sql (hunk 1 above)
Comment (af-afk) [reply database/102-up-timescale/20230502151630-pending_reward_type_add_log_index.sql]: Why not decimal/number?

Comment (eli-d) [inline database/102-up-timescale/20230502151630-pending_reward_type_add_log_index.sql]: non nullable

File: database/102-up-timescale/20230502154725-user_actions_add_log_index.sql [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+	ADD COLUMN log_index BIGINT;
```
Comment (eli-d) [inline database/102-up-timescale/20230502154725-user_actions_add_log_index.sql]: non nullable

**Review by af-afk — CHANGES_REQUESTED** [review]

**Review by eli-d — COMMENTED** [review]
also need to fix tests

**Review by af-afk — APPROVED** [review]

**Pull request merged**